lightning/src/routing/network_graph.rs

```diff
@@ -2403,6 +2425,17 @@ mod tests {
 					assert_eq!(msg.number_of_blocks, expected_reply.number_of_blocks);
 					assert_eq!(msg.sync_complete, expected_reply.sync_complete);
 					assert_eq!(msg.short_channel_ids, expected_reply.short_channel_ids);
+
+					// Enforce exactly the sequencing requirements present on c-lightning v0.9.3
+					assert!(msg.first_blocknum == c_lightning_0_9_prev_end_blocknum || msg.first_blocknum == c_lightning_0_9_prev_end_blocknum.saturating_add(1));
+					assert!(msg.first_blocknum >= max_firstblocknum);
```

Comment:
* Since we're checking for `>=`, wouldn't it be `min_firstblocknum`? 
* Is it fair to say that `max_firstblocknum` is checking that the `msg`'s first_blocknum is at least the first_blocknum of the previous `msg`?

Reply:
Its the maximum-seen-firstblocknum, which each new message must be greater than. 

> Is it fair to say that max_firstblocknum is checking that the msg's first_blocknum is at least the first_blocknum of the previous msg?
Yes